docs/understanding-airbyte/connections/README.md

```diff
@@ -21,10 +16,46 @@ Syncs will be triggered by either:
 
 When a scheduled connection is first created, a sync is executed as soon as possible. After that, a sync is run once the time since the last sync \(whether it was triggered manually or due to a schedule\) has exceeded the schedule interval. For example, consider the following illustrative scenario:
 
-* **October 1st, 2pm**, a user sets up a connection to sync data every 24 hours. 
-* **October 1st, 2:01pm**: sync job runs 
-* **October 2nd, 2:01pm:** 24 hours have passed since the last sync, so a sync is triggered. 
+* **October 1st, 2pm**, a user sets up a connection to sync data every 24 hours.
+* **October 1st, 2:01pm**: sync job runs
+* **October 2nd, 2:01pm:** 24 hours have passed since the last sync, so a sync is triggered.
 * **October 2nd, 5pm**: The user manually triggers a sync from the UI
 * **October 3rd, 2:01pm:** since the last sync was less than 24 hours ago, no sync is run
 * **October 3rd, 5:01pm:** It has been more than 24 hours since the last sync, so a sync is run
 
+## Destination namespace
+
+The location of where a connection replication will store data is referenced as the destination namespace. The destination connectors should create and write records (for both raw and normalized tables) in the specified namespace which should be configurable in the UI thanks to the Namespace Configuration field (or NamespaceDefinition in the API). You can read more about configuring namespaces [here](../namespaces.md).
+
+## Destination stream name
+
+### Prefix stream name
+
+Stream names refers to table names in a typical RDBMS. But it can also be the name of an API endpoint etc. Similarly to the namespace, stream names can be configured to diverge from their names in the source with a "prefix" field. The prefix is prepended to the source stream name in the destination.
```

Comment:
```suggestion
Stream names refer to table names in a typical RDBMS. But it can also be the name of an API endpoint, etc. Similarly to the namespace, stream names can be configured to diverge from their names in the source with a "prefix" field. The prefix is prepended to the source stream name in the destination.
```